articles/language/type-model.md

```diff
@@ -159,6 +123,22 @@ newtype DoublyWrappedInt = WrappedInt;
 ...
 ```
 
+User-defined types may be used anywhere any other type may be used.
+In particular, it is possible to define an array of a user-defined type
+and to include a user-defined type as an element of a tuple type.
+
+It is not possible to create recursive type structures.
+That is, the type that defines a user-defined type may not be a tuple type
+that includes an element of the user-defined type.
+More generally, user-defined types may not have cyclic dependencies
+on each other, so the following set of type definitions would be illegal:
+
+```qsharp
+newtype TypeA = (Int, TypeB);
```

Comment:
Perhaps annotating this line with a comment indicating the error would help prevent someone skimming through from accidently thinking it's valid code:

```suggestion
newtype TypeA = (Int, TypeB); // error: TypeB has not been defined.
```

Reply:
the sample illustrates a cyclic dependency (ie the suggested comment would be incorrect) and that is stated in the sentence above